dist/mastodon-streaming@.service

```
@@ -0,0 +1,54 @@
+[Unit]
+Description=mastodon-streaming on port %I
+After=network.target
+# handles using `systemctl restart mastodon-streaming`
+PartOf=mastodon-streaming.service
+ReloadPropagatedFrom=mastodon-streaming.service
+
+[Service]
+Type=simple
+User=mastodon
+WorkingDirectory=/home/mastodon/live
+Environment="NODE_ENV=production"
+Environment="PORT=%i"
+ExecStart=/usr/bin/node ./streaming
```

This should continue to work post the merge of #24702, but we may wish to update this to:
```suggestion
WorkingDirectory=/home/mastodon/live/streaming
ExecStart=/usr/bin/node ./index.js
```

Such that it's inline with how streaming expects to be run based on the package.json